src/components/PoolingWidget/DefineSpread.tsx

```diff
@@ -43,6 +43,10 @@ export const SpreadInformation: React.FC<SpreadInformationProps> = ({ selectedTo
             &::before {
               content: none;
             }
+          }
+          > td {
+            padding: 0.7rem 0 !important;
+            margin: 0 0.6rem !important;
```

Comment:
I generally don't like `!important`, but I trust you.
Maybe raise specificity with something like a class

Reply:
i agree here